lib/logger.js

```diff
@@ -1,23 +1,19 @@
 'use strict'
-const winston = require('winston')
+const {createLogger, format, transports} = require('winston')
 
-class Logger extends winston.Logger {
-  // Implement stream.writable.write interface
-  write (chunk) {
-    this.info(chunk)
-  }
-}
 
-module.exports = new Logger({
+module.exports = createLogger({
+  format: format.combine(
+    format.uncolorize(),
+    format.timestamp(),
+    format.align(),
+    format.splat(),
+    format.printf(info => `${info.timestamp} ${info.level}: ${info.message}`)
+  ),
   transports: [
-    new winston.transports.Console({
+    new transports.Console({
       level: 'debug',
```

Comment:
I would really love to get rid of the debug logging as default, but since we use this logging class in the config module, we shouldn't include the config here. So I'm not sure how to solve this.